### PR TITLE
Adding route

### PR DESCRIPTION
**Pull request by bcgodfrey91**
## Purpose

Needed to create a main route of '/' that would render 5 reminders from storage. Had to create the model, route, and template for these.

## Approach

This change has our first test passing, signaling that this task should be complete.

### Learning

For research, we looked at previous repos we have worked on. Specifically, these two: 
1) https://github.com/turingschool-examples/beard-beats-two
2) https://github.com/turingschool-examples/ember-groceries

From there, we modeled our set up after these two.

### Open Questions and Pre-Merge TODOs

- [ ] Use Github checklists. When solved, check the box and explain the answer.

### Test coverage 

We used the test that was given to us. It is testing to make sure that 1) we have an initial route of '/', and 2) we have 5 reminders rendering onto the screen.

### Merge Dependencies and Related Work

None

### Follow-up tasks

None

#### ITS SO LIT
<iframe src="//giphy.com/embed/mtaWx98w7mX7y" width="480" height="360" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/pikachu-nyan-cat-pika-mtaWx98w7mX7y">via GIPHY</a></p>

### CODE REVIEW

File: app/models/reminder.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  title: DS.attr('string'),
+  date: DS.attr('date'),
+  notes: DS.attr('string'),
+  pinned: DS.attr('boolean')
```
Comment (stevekinney) [inline app/models/reminder.js]: Could we add a default value of `false`?

File: app/router.js [hunk 2]
```diff
@@ -7,6 +7,9 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
+  this.route('reminders', { path: '/' }, function() {
+    this.route('reminder', { path: ':id'});
```
Comment (stevekinney) [inline app/router.js]: We don't need this route just yet.

File: app/routes/reminders/reminder.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
```
Comment (stevekinney) [inline app/routes/reminders/reminder.js]: We don't need this yet. I'd remove this route until the next user story.

File: app/templates/index.hbs [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<div class="index template">
+  <p>This is <em>index.hbs</em>.</p>
```
Comment (stevekinney) [inline app/templates/index.hbs]: I get that this is the `index` template, but we don't need to display that to the user.

File: app/templates/reminders.hbs [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<div class="artists template">
```
Comment (stevekinney) [inline app/templates/reminders.hbs]: Artists template?

**Comment by bcgodfrey91** [comment]
@stevekinney @martensonbj we've made the suggested changes. Let us know what you think.

File: app/models/reminder.js [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import DS from 'ember-data';
+
+export default DS.Model.extend({
+  title: DS.attr('string'),
+  date: DS.attr('date'),
+  notes: DS.attr('string'),
+  pinned: DS.attr(false)
```
Comment (martensonbj) [inline app/models/reminder.js]: Check out [the docs](https://guides.emberjs.com/v2.8.0/models/defining-models/#toc_options) for how to implement the default value correctly

File: app/templates/application.hbs [hunk 7]
```diff
@@ -0,0 +1,5 @@
+<div class="application template">
+  <h1>Welcome to Remember</h1>
+  <p>This is <em>application.hbs</em>.</p>
```
Comment (martensonbj) [inline app/templates/application.hbs]: User doesn't need to see this line.

File: app/templates/reminders.hbs [hunk 8]
```diff
@@ -0,0 +1,11 @@
+<div class="reminders-template">
+  <p>This is <em>reminder.hbs</em>.</p>
+
+  <ul class="reminders">
```
Comment (martensonbj) [inline app/templates/reminders.hbs]: Not the best semantic tag for this - Go with div/p or section/article.

**Comment by bcgodfrey91** [comment]
How does this look @martensonbj? We shouldn't need to do the same with date since those are already stored, correct?

File: app/templates/index.hbs [hunk 9]
```diff
@@ -0,0 +1,3 @@
+<div class="index template">
+  {{outlet}}
+</div>
```
Comment (martensonbj) [inline app/templates/index.hbs]: This file isn't necessary

**Comment by martensonbj** [comment]
Cool! :smile:

**Pull request merged**